### PR TITLE
 [Check-in] Add guide for "EGI Check-in Token Portal"

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

Add "EGI Check-in Token Portal" guide in the new section "Users"<"Authentication & Authorization"<"EGI Check-in"<"Obtaining Tokens".
Also, all the references to EGI Fedcloud Token Portal have been fixed

### CODE REVIEW

**Comment by gwarf** [comment]
Dear @vardizzo-lab please review and let us know when this PR can be merged, the preview will be soon available at https://docs.egi.eu/documentation/446

File: content/en/users/aai/check-in/obtaining-tokens/token-portal/_index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+description: >
+  Usage guide of EGI Check-in Token Portal
+---
+
+The EGI Check-in Token Portal allows users to create Access and Refresh Tokens.
+
+## Obtain an Access Token
+
+In order to obtain an Access Token from EGI Check-in Token Portal, please follow
+the steps bellow:
```
Comment (vardizzo-lab) [inline content/en/users/aai/check-in/obtaining-tokens/token-portal/_index.md:17]: below.

File: content/en/users/aai/check-in/obtaining-tokens/token-portal/_index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   1. The value of the Access Token
+   2. The command to get user's information from userinfo endpoint
+   3. Obtain a Refresh Token (more info in
+      [Obtain a Refresh Token](#obtain-a-refresh-token) section)
+   4. Manage your active Access/Refresh Tokens
+
+## Obtain a Refresh Token
+
+In order to obtain an Refresh Token from EGI Check-in Token Portal, please
+follow the steps bellow:
```
Comment (vardizzo-lab) [inline content/en/users/aai/check-in/obtaining-tokens/token-portal/_index.md:32]: below

**Review by vardizzo-lab — CHANGES_REQUESTED** [review]

File: content/en/users/aai/check-in/obtaining-tokens/token-portal/_index.md (hunk 1 above)
Comment (nikosev) [reply content/en/users/aai/check-in/obtaining-tokens/token-portal/_index.md]: typo fixed

File: content/en/users/aai/check-in/obtaining-tokens/token-portal/_index.md (hunk 2 above)
Comment (nikosev) [reply content/en/users/aai/check-in/obtaining-tokens/token-portal/_index.md]: typo fixed

**Review by vardizzo-lab — APPROVED** [review]

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**